target/generated-sources/xjc/META-INF/sun-jaxb.episode

```diff
@@ -0,0 +1,95 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
please exclude generated sources from git and provide readme how to generate them ;-)